extensions/micrometer-opentelemetry/deployment/src/main/java/io/quarkus/micrometer/opentelemetry/deployment/MicrometerOTelBridgeConfigBuilderCustomizer.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.micrometer.opentelemetry.deployment;
+
+import java.util.Map;
+
+import io.smallrye.config.PropertiesConfigSource;
+import io.smallrye.config.SmallRyeConfigBuilder;
+import io.smallrye.config.SmallRyeConfigBuilderCustomizer;
+
+public class MicrometerOTelBridgeConfigBuilderCustomizer implements SmallRyeConfigBuilderCustomizer {
+    @Override
+    public void configBuilder(final SmallRyeConfigBuilder builder) {
+        // use a priority of 50 to make sure that this is overridable by any of the standard methods
+        builder.withSources(
+                new PropertiesConfigSource(Map.of(
+                        "quarkus.otel.metrics.enabled", "true"),
+                        "quarkus-micrometer-opentelemetry", 1),
```

Comment:
What is `quarkus-micrometer-opentelemetry` and why is the value `1`?

Reply:
It's a mistake. Will fix.

Reply:
Actually, it's right but I will merge it with the next source.
`quarkus-micrometer-opentelemetry` is the config source name and 1 is the priority.